site/content/contribute/gsoc/_index.md

```diff
@@ -3,224 +3,58 @@ title: "Google Summer of Code"
 description: "Ideas list for Google Summer of Code"
 ---
 
-# GSoC 2021 Ideas
+# Google Summer of Code
+This page outlines everything you need to know about contributing to Mattermost for the Google Summer of Code (GSoC).
```

Comment:
Do we need the article? Not sure, actually, so this is more a question than a request for changes :)
```suggestion
This page outlines everything you need to know about contributing to Mattermost for Google Summer of Code (GSoC).
```